testSet.ts

```diff
@@ -44,12 +44,28 @@ function getTestsetDirectories(): string[] {
         .map((name) => path.join(testsetsPath, name));
 }
 
-async function loadTestObject(filename: string): Promise<TestCase> {
+async function loadTestObject(filename: string): Promise<ValidTestCase> {
     try {
         const module = await import(filename);
-        return module.default; // Access the default export
+        return validateTestCase(filename, module.default); // Access the default export
     } catch (error) {
         console.error(`Failed to load module: ${filename}`, error);
         throw error;
     }
 }
+function validateTestCase(testCaseFileName: string, testCase: TestCase): ValidTestCase {
+    return { testCaseFileName, testCase };
+}
+
+export function getAbsoluteFileResultFromSQLResult(result: SQLResult, testCaseFileName: string): AbsoluteFileResult {
+    let fileName = testCaseFileName + '.result.ndjson';
```

Comment:
Would it be possible to make those json files? I would prefer that, since LAPIS also returns JSON. From reading the expected results one gets the impression that LAPIS would return NDJSON.

-> I would like to have the expected results as close as possible to the actual results.

```suggestion
    let fileName = testCaseFileName + '.result.json';
```

Reply:
duckdb is unfortunately not able to generate these..

Reply:
I also tried around, but that is the reason, why I had to add the extra abstraction

Reply:
Ok... Maybe later then.